client/src/components/ReportTable.js

```diff
@@ -10,6 +10,7 @@ import moment from "moment"
 import PropTypes from "prop-types"
 import React, { useEffect, useRef, useState } from "react"
 import { Table } from "react-bootstrap"
+import PlanningConflictForReport from "components/PlanningConflictForReport"
```

Comment:
Please properly order the imports (alphabetically on imported package if there's no command for it in your IDE).